assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -55,12 +55,15 @@ function DisplayElement( { element } ) {
 		if ( transform === null ) {
 			target.style.transform = '';
 		} else {
-			const { translate, rotate, resize } = transform;
+			const { translate, rotate, resize, fontSize } = transform;
 			target.style.transform = `translate(${ translate[ 0 ] }px, ${ translate[ 1 ] }px) rotate(${ rotate }deg)`;
 			if ( resize[ 0 ] !== 0 && resize[ 1 ] !== 0 ) {
 				target.style.width = `${ resize[ 0 ] }px`;
 				target.style.height = `${ resize[ 1 ] }px`;
 			}
+			if ( fontSize && target.firstChild ) {
+				target.firstChild.style.fontSize = fontSize + 'px';
```

Comment:
Does it have to be set on `.firstChild`? Can't it inherit from parent or does it have another default?

Reply:
I think instead of `firstChild` it'd cleaner to use the `useTransformHandler` inside the element's display or frame template to do the math. The image display does something similar. The display/frame templates have a lot more knowledge about the structure of the element and properties that could change.

Reply:
Makes sense to look into something else than relying on `firstChild`. Currently, the font calculation is also using `firstChild` of the target.

Note that I can look into this after time off, alternatively, feel free to take over.